.github/workflows/sanity_checks_utcs.py

```diff
@@ -0,0 +1,264 @@
+import json
+import os
+import re
+
+CORRECT_UNICODE = "\u2713"
+WRONG_UNICODE = "\u2715"
+CONNECTOR_CATEGORY = ['Analytics and SIEM', 'Authentication', 'Case Management', 'Threat Intelligence', 'Database',
```

Comment:
can be part of another constants file

Reply:
Yes, We can do this. But in this scenario single file script will be better IMO.